lib/locale/pl_PL/strings.php

```diff
@@ -1886,7 +1887,6 @@
 $_LANG['SMTP settings.'] = 'Ustawienia SMTP.';
 $_LANG['Software version:'] = 'Wersja oprogramowania:';
 $_LANG['Sorting:'] = 'Sortowanie:';
-$_LANG['Sorting'] = 'Sortowanie';
```

Comment:
Usuwasz dzisiejszą poprawkę? Gdzie niegdzie już `Sorting` jest używane w UI.

Reply:
Za to przepraszam - wkradło się jakoś jak robiłem rebase. Cofnąłem w https://github.com/chilek/lms/pull/2273/commits/0c2dc742eb5ea75af4830d164c4ffe287a04cdc0